var/spack/repos/builtin/packages/g2/package.py

```diff
@@ -38,25 +38,28 @@ class G2(CMakePackage):
         when="@3.4.6:",
     )
     variant("w3emc", default=True, description="Enable GRIB1 through w3emc", when="@3.4.6:")
-    variant("shared", default=False, description="Build shared library", when="@3.4.7:")
-    variant("openmp", default=False, description="Use OpenMP multithreading", when="@develop")
-    variant("utils", default=False, description="Build grib utilities", when="@develop")
+    variant("aec", default=True, description="Use AEC library", when="@4:")
```

Comment:
The `shared` variant has been deleted, but mentions of it are still in the recipe.

Reply:
I must have removed by accident. Just restored.